### ISSUE TITLE
Crash importing/using image sequences

### ISSUE THREAD

**Issue by leodf1** [OpenShot/openshot-qt#5410]
**Describe the bug:**
Importing and/or using an image sequence, crashes to desktop. Either immediatelly on import, on playback of the sequence in the timeline

**Steps to reproduce the behavior:**
1. Import one image of the sequence
2. Program prompts to Import as Image sequence
3. Click 'Yes'
4. [Images displayed as single sequence in Project Files]
5. Drag Image sequence onto timeline
6. Playback the sequence in the timeline
7. Crash to desktop

**Expected behavior:**

**System Details:**
- OpenShot Version [e.g. 2.4.3]: {{ log_context.openshot_qt_version }}
- Operating System / Distro: [e.g. Windows 10, Linux Mint 17.1]

**Log Files:**
If you are experiencing a crash, please collect and attach logs of the problem.  Log files can be found in the `.openshot_qt` folder in your user home directory. Log files over 2 MB in size will need to be compressed, please attach a ZIP file instead of the raw logs.

**Exception / Stacktrace:**
If you are experiencing a crash or strange output on the command line, please attach a small snippet of the exception here, but please keep it as small as possible (for readability).

**Screenshots:** (Optional)
If applicable, add screenshots to help explain your problem. To include screenshots, copy/paste them here, drag the image(s) into this form, or click the link below this edit field to load images saved to your computer. All images are public, so please don't post screenshots containing personal information.

[Uploading Testimage 000.zip…]()

**Comment by leodf1** [comment]
I have provided a set of 3 images in the Screenshots section above. Please open a new project and import one image. You will be prompted to import the set as an Image Sequence. Accept, then place the sequence on the timeline and attempt to play it.

This causes a crash for me.

Usually I get the crash on import of image sequences, but for this set, the crash happens on playback...

Let me know, thanks

**Comment by Colorjet3** [comment]
Hello @leodf1.  When I click on the zip file nothing is happening.  Please re-upload and I will test.

**Comment by leodf1** [comment]
Try here:
[Testimage 000.zip](https://github.com/OpenShot/openshot-qt/files/14200670/Testimage.000.zip)

**Comment by Colorjet3** [comment]
Hello @leodf1.  I was able to replicate the issue of OpenShot crashing.  This has been reported and worked on previously but was never fully vetted.  I did some additional testing and ran into a few other issues as well so I am going to come back to this ticket once I have some time to do some further testing and report my findings here.  Stay tuned....

**Update**:  Found this - https://github.com/OpenShot/openshot-qt/issues/5321

**Comment by leodf1** [comment]
Thank you

I did find that before posting. But the technicalities prevented me from figuring out if it was the exact same issue, or just related. Decided to post anyway, just in case....

**Comment by Colorjet3** [comment]
Yeah, I don't think it is exactly the same.  I just wanted you to be aware of it.  I'll start my testing in the next few days.

**Comment by James763** [comment]
I am experiencing the same crashing issue when I add a second instance of the same "popcorn in a speaker box" background image to my timeline and expand its length with my resize cursor to fit the rest of the length of a movie as a background image for one of my album/movie sync video projects, which is of the 4K HDR version of _Willy Wonka & the Chocolate Factory_ (1971) synced with _Primus & the Chocolate Factory with the Fungi Ensemble_ by Primus (2014). The project is in 2160p at 23.98fps and I have resized the 3840x2076 (1.85:1) movie MKV file to 1920x1038 and moved it to fit in the empty part of the background image.

**Comment by Colorjet3** [comment]
Hello @James763 Does your issue have to do with "Image Sequences"?  If not, you should start a new thread.
